src/ResourceManager/Sql/Commands.Sql.Test/packages.config

```diff
@@ -17,6 +17,8 @@
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.14" targetFramework="net452" />
```

Comment:
@ziwa-msft same comment about removing the references to ClientRuntime in this file